src/runtime_src/core/tools/common/ReportCu.cpp

```diff
@@ -251,14 +254,23 @@ ReportCu::getPropertyTree20202( const xrt_core::device * _pDevice,
 }
 
 void 
-ReportCu::writeReport( const xrt_core::device* /*_pDevice*/,
+ReportCu::writeReport( const xrt_core::device* _pDevice,
                        const boost::property_tree::ptree& _pt, 
                        const std::vector<std::string>& /*_elementsFilter*/,
                        std::ostream & _output) const
 {
   boost::property_tree::ptree empty_ptree;
+  auto uuid = xrt::uuid(xrt_core::device_query<xrt_core::query::xclbin_uuid>(_pDevice));

```

Comment:
Please add this in `getPropertyTree20202` ptree so that the json is also updated. You can then call `pt.get(compute_units.xclbin_uuid)` to retreive the UUID